.github/workflows/ci.yml

```
@@ -725,15 +727,10 @@ jobs:
         run: |
           cp .github/cabal.project cabal.project.local
 
-      - name: Download artifact _build directory
-        uses: actions/download-artifact@v4
-        with:
-          name: _build-${{ matrix.target.top }}
-          path: _build
-
       - run: cache pull cabal
       - run: cache pull cargo
       - run: cache pull build
+      - run: cache pull build-post-synth --prefix="${{ matrix.target.top }}"
```

Can you add a name, so it looks better in GitHub Actions? (change the name to your liking)
```suggestion
      - name: cache pull bittide-post-synth
        run: cache pull build-post-synth --prefix="${{ matrix.target.top }}"
```

---

yessir